aptos-move/aptos-vm/src/move_vm_ext/vm.rs

```diff
@@ -73,6 +74,7 @@ impl MoveVmExt {
 
         extensions.add(NativeTableContext::new(txn_hash, remote));
         extensions.add(NativeRistrettoPointContext::new());
+        extensions.add(ArksContext::new());
```

Comment:
Probably call it `EllipticCurvesContext`, which is independent of the library name we chose to implement with.